src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -148,7 +152,7 @@
  * <i>Relation:</i>
  *         "#" / "&lt;" / "&le;"
  * <i>Format:</i>
- *         {@code String}
+ *         Any unicode characters except the <i>Relation</i> symbols
```

Comment:
Nit: since characters in Java are of Unicode code points, no need to mention `unicode` here.

Reply:
Thanks for the follow-up review. Fixed the nit, and updated the [CSR](https://bugs.openjdk.org/browse/JDK-8314546) to reflect all of the changes.

(I used the 'Unicode' wording because the other Format classes did, I will adjust it for the other classes in this [issue](https://bugs.openjdk.org/browse/JDK-8315946)